ros2bag/ros2bag/verb/play.py

```diff
@@ -88,6 +88,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--disable-keyboard-controls', action='store_true',
             help='disables keyboard controls for playback')
+        parser.add_argument(
+            '-p', '--start-paused', action='store_true', default=False,
+            help='Start the playback player paused.')
```

Comment:
Would it be a bit more clearer if the help text was re-worded as "Start the playback player in a paused state." ? Minor quibble, of course, but I could see some confusion for people not familiar with what this option does.

Reply:
Both options sound the same to me, though I'm not a native speaker.
Fixed in https://github.com/ros2/rosbag2/pull/904/commits/4ff72610999249babfabf2b97ae1d1cb177182bf.